tests/test_logger.py

```diff
@@ -71,206 +71,226 @@ def shutdown(self, wait=True):
 
 
 class LogDNAHandlerTest(unittest.TestCase):
-    def setUp(self):
```

Comment:
Most of the changes in this class are due to the removal of the setup / teardown of the unit test class instance of the handler. This is no longer necessary since we are removing the creation of the flusher thread from the lifecycle of the class itself. It also is inconvenient since it gets in the way of some of the mock patching that is used in the unit tests.